lib/blurb/report_requests.rb

```diff
@@ -2,29 +2,31 @@
 
 class Blurb
   class ReportRequests < RequestCollectionWithCampaignType
-    SD_TACTIC = 'T00020'.freeze
 
     def initialize(campaign_type:, base_url:, headers:)
       @campaign_type = campaign_type
       @base_url = "#{base_url}/v2/#{@campaign_type}"
       @headers = headers
+      @tactic = "T00020"
```

Comment:
We need to pass it as a parameter not hardcode it to "T00020".

Reply:
@rrails I think `T00020` should be the default for backwards compatibility and we use the passed `tactic` param if present

Reply:
It get pass in to create method, in initialize I set it to a default value.

Reply:
Use param if present. So we should add the param now?

Reply:
I now understand - all good to go.